collectors/python.d.plugin/varnish/README.md

```diff
@@ -29,16 +30,15 @@ This module produces the following charts:
 -   Backend Connections Statistics in `connections/s`
 -   Requests To The Backend in `requests/s`
 -   ESI Statistics in `problems/s`
--   Memory Usage in `MiB`
```

Comment:
Why do you delete `Memory Usage` chart from the list?

Reply:
Upss! It seems this chart does not appear in Varnish-Plus (probably another incompatibility). I will take a look and create another pull request for fixing that. Let me put this chart back in the documentation. Thanks!

Reply:
Done. I put the chart back in the list.